### ISSUE TITLE
Add pause buttons to the UI

### ISSUE THREAD

**Issue by justinross** [ThatOdieGuy/SeedboxSync#25]
One near the top for "pause all transfers", and one on each in-progress transfer to pause that specific transfer.

**Comment by justinross** [comment]
Probably requires #5

**Comment by ThatOdieGuy** [comment]
Yeah, this one is going to need some work into file resume. I'll investigate how long that will take.
